css/properties/stroke-color.json

```
@@ -24,7 +24,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤12.1"
```

```suggestion
              "version_added": "11.1"
```

See: https://github.com/caugner/css-properties/commit/b5ec7ac5240529cb159b62ac198b46f58406fa09#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R298